README.md

```diff
@@ -54,6 +54,9 @@ const sourceCode = 'const {foo} = bar'
 const options = {
   text: sourceCode,
   eslintConfig: {
+    parserOptions: {
+      ecmaVersion: 7,
```

Comment:
This is the reason that you got `undefined` back. Because ESLint actually failed to parse the code (because it used `const`).